chia/farmer/farmer.py

```diff
@@ -637,7 +641,7 @@ async def get_cached_harvesters(self, connection: WSChiaConnection) -> Harvester
             self.harvester_cache[connection.peer_host] = host_cache
         node_cache = host_cache.get(connection.peer_node_id.hex())
         if node_cache is None:
-            node_cache = HarvesterCacheEntry()
+            node_cache = HarvesterCacheEntry(self.update_harvester_cache_interval)
```

Comment:
this seems to be the main use of this member. You don't think it would be simpler and more direct to just pass this value as a parameter to this function? (i.e. to `get_cached_harvesters()`)